packages/color-picker/src/index.vue

```diff
@@ -0,0 +1,256 @@
+          <span class="el-color-dropdown__value">
+            <input
+              v-model="customInput"
+              :validate-event="false"
+              size="mini"
+              @keyup.enter="handleConfirm"
+              @blur="handleConfirm"
+            >
+          </span>
+          <el-button
+            size="mini"
+            type="text"
+            class="el-color-dropdown__link-btn"
+            @click="clear"
+          >
+            {{ t('el.colorpicker.clear') }}
+          </el-button>
+          <el-button
+            plain
+            size="mini"
+            class="el-color-dropdown__btn"
+            @click="confirmValue"
+          >
+            {{ t('el.colorpicker.confirm') }}
+          </el-button>
+        </div>
+      </div>
+    </template>
+    <template #default>
+      <div
+        :class="[
+          'el-color-picker',
+          colorDisabled ? 'is-disabled' : '',
+          colorSize ? `el-color-picker--${ colorSize }` : ''
+        ]"
+      >
+        <div v-if="colorDisabled" class="el-color-picker__mask"></div>
+        <div class="el-color-picker__trigger" @click="handleTrigger">
+          <span class="el-color-picker__color" :class="{ 'is-alpha': showAlpha }">
+            <span
+              class="el-color-picker__color-inner"
+              :style="{
+                backgroundColor: displayedColor
+              }"
+            ></span>
+            <span v-if="!modelValue && !showPanelColor" class="el-color-picker__empty el-icon-close"></span>
+          </span>
+          <span v-show="modelValue || showPanelColor" class="el-color-picker__icon el-icon-arrow-down"></span>
+        </div>
+      </div>
+    </template>
+  </el-tooltip>
+</template>
+
+<script lang="ts">
+import { defineComponent,computed,ref,nextTick,reactive,watch,provide,inject } from 'vue'
+import ClickOutside from '@element-plus/directives/click-outside'
+import Color from './color'
+import SvPanel from './components/sv-panel.vue'
+import HueSlider from './components/hue-slider.vue'
+import AlphaSlider from './components/alpha-slider.vue'
+import ElTooltip from '@element-plus/tooltip/src/index.vue'
+import ElButton from '@element-plus/button/src/button.vue'
+import { t } from '@element-plus/locale/index'
+import { UPDATE_MODEL_EVENT }  from '@element-plus/utils/constants'
+import Predefine from './components/predefine'
+// import ElPopper from '@element-plus/popper/src/index.vue'
+
+interface IELEMENT {
+  size?: string
+}
+interface IElForm {
+  size?: string
+  disabled?: boolean
+}
+interface IELFormItem {
+  elFormItemSize?: string
+}
+
+const SY = Symbol()
+
+export const useOptions = () => {
+  return inject(SY)
+}
+
+export default defineComponent( {
+  name: 'ElColorPicker',
+  components: {
+    // ElPopper,
+    ElTooltip,
+    SvPanel,
+    HueSlider,
+    AlphaSlider,
+    ElButton,
+    Predefine,
+  },
+  directives: {
+    ClickOutside,
+  },
+  props: {
+    modelValue: String,
```

Comment:
modelValue will not be reactively any where?
i saw the element-ui code, they watch value's change

Reply:
done